voice/bxml/callbacks/redirect.md

```diff
@@ -62,6 +64,7 @@ POST http://[External server URL]
 ```json
 {
 	"eventType"     : "redirect",
+	"eventTime"     : "2019-06-20T15:54:22.807Z",
```

Comment:
Not sure it's possible to get a `redirect` event before `answerTime`, but it's kinda weird anyway.